### ISSUE TITLE
The https://sembr.org website appears to be down

### ISSUE THREAD

**Issue by RohanTalip** [sembr/specification#10]
![Screen Shot 2022-08-16 at 12 48 16 PM](https://user-images.githubusercontent.com/7445140/184971720-eaa6294c-ca1c-4764-ade5-11df684de1c0.png)

**Comment by waldyrious** [comment]
Seems to be working for me at the time of writing. Maybe it got restored in the meantime?

**Comment by xgpt** [comment]
Have you considered hosting what appears to be a static website with github pages? It is free and can even utilize your sembr.org domain name! Please let me know if you'd like assistance in transitioning to that, it would save you hosting costs and leverages the stability of GitHub to ensure that there's far less downtime. Did I mention *free*?

**Comment by waldyrious** [comment]
Just to be clear, I wasn't the reporter of this issue — @RohanTalip was :)

(Though I'll take the opportunity to subscribe to @xgpt's suggestion — why not host in GitHub pages?)

**Comment by waldyrious** [comment]
Right, that's a fair point. But on the other hand, using GitHub Pages would allow this project to be more collaborative, since people could contribute to the website by merely editing files in the repo (it seems to be essentially the README minus the "Summary" section heading, right?), without requiring an extra deploy/sync step.

It's not a big deal, of course, just a set up I personally find more appealing for open source projects.
